lib/web/tty_playback.go

```diff
@@ -53,6 +53,47 @@ const (
 	actionPause = byte(1)
 )
 
+func (h *Handler) sessionLengthHandle(
+	w http.ResponseWriter,
+	r *http.Request,
+	p httprouter.Params,
+	sctx *SessionContext,
+	site reversetunnelclient.RemoteSite,
+) (interface{}, error) {
+	sID := p.ByName("sid")
+	if sID == "" {
+		return nil, trace.BadParameter("missing session ID in request URL")
+	}
+
+	ctx, cancel := context.WithCancel(r.Context())
+	defer cancel()
+
+	clt, err := sctx.GetUserClient(ctx, site)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	evts, errs := clt.StreamSessionEvents(ctx, session.ID(sID), 0)
```

Comment:
> since the web UI is making two requests that both read the session recording, the audit log will show two separate session_recording.access events.

could we prevent this by modifying `StreamSessionEvents()` to accept an `...opts` param with an optional `noAuditEvent` flag that we can pass in here which will let it know to skip the audit event?

Reply:
We could, but I don't really want to make it easy to do things without being audited.